docs/proposals/20230407-optional-cluster-infra-resource.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Make Cluster Infra Resource Optional
+authors:
+  - "@jackfrancis"
+reviewers:
+  - "@richardcase"
+  - "@pydctw"
+  - "@mtougeron"
+  - “@CecileRobertMichon”
+  - “@fabriziopandini”
+  - “@sbueringer”
+  - "@killianmuldoon"
+  - "@mboersma"
+  - "@nojnhuh"
+creation-date: 2023-04-07
+last-updated: 2023-04-07
+status: provisional
+see-also:
+  - "/docs/proposals/20220725-managed-kubernetes.md"
+---
+
+# Make Cluster Infra Resource Optional
```

Comment:
If indeed we want to support > 1 scenario, then I agree this title is insufficient. I have a slight preference for making the title reflect the specific changes proposed, rather than something high level like "Managed Kubernetes Changes". Definitely open to ideas here.

Reply:
Went with Richard's original idea. :)